packages/client/src/views/SysAdmin/Config/Forms/Home/ActionsNotification.tsx

```diff
@@ -35,24 +36,25 @@ export function ActionsNotification() {
   )
 
   return (
-    <Toast
-      type={
-        isNotifiable(status)
-          ? NOTIFICATION_TYPE_MAP[status]
-          : NOTIFICATION_TYPE.ERROR
-      }
-      show={isNotifiable(status)}
-      callback={
-        status !== ActionStatus.PROCESSING
-          ? () => setAction({ status: ActionStatus.IDLE })
-          : undefined
-      }
-    >
-      {isNotifiable(status) &&
-        intl.formatMessage(statusChangeActionMessages(action)[status], {
-          event: intl.formatMessage(constantsMessages[event]),
-          version
-        })}
-    </Toast>
+    <>
+      {isNotifiable(status) && (
+        <Toast
+          type={
+            isNotifiable(status) ? NOTIFICATION_TYPE_MAP[status] : 'warning'
```

Comment:
I think we can safely remove the `isNotifiable(status)` check from here as we are doing so before calling the Toast. Nice job removing the `show` prop from the `Toast` component. It made some of the checks like this not needed anymore.